### ISSUE TITLE
Boxplot border disappeared, strange color as outliers added

### ISSUE THREAD

**Issue by degill** [mwaskom/seaborn#787]
Hey there,

I have a pretty strange behaviour, but I dont know when exactly it started. 

When I issue the following commands:
```
    # custom color
    pal = [mygreen, myblue]
    # Start a subplot
    fig, axre = plt.subplots(1, figsize=(4,5))
    # Plot the actual data on the subplot we created
    with sns.axes_style('whitegrid'):
        plot = sns.boxplot(x=real_experiment, y="error_degree", data=dataset_frame, palette=pal, ax=axre)
```

I get the following result:

![image](https://cloud.githubusercontent.com/assets/6094292/11457020/7fdbba6a-969d-11e5-81c6-70cc505f055c.png)

I dont know why the outliers are now marked as blue (which is a different blue than myblue) and why the box around the boxplot box disappeared.

Seaborn Version: 0.7.0.dev0 (but I also have the issue with the non-dev version)
Matplotlib: 1.5.0

Any ideas?

**Comment by mwaskom** [comment]
Python 3?

**Comment by degill** [comment]
Yes I use Python3

**Comment by mwaskom** [comment]
Sounds like a (known) matplotlib bug that affects the outline colors of patch objects, but I'm not sure.

**Comment by jrderuiter** [comment]
I have the same issue in Python 3 (3.5), with the current (non-dev) version of Seaborn and matplotlib 1.5.0. Has this been fixed in newer matplotlib versions (2.0?)?

**Comment by mwaskom** [comment]
If this is the underlying issue, then yes: https://github.com/matplotlib/matplotlib/issues/5423

However I'm not sure it's going to be in 2.0, that is a "style change only" release, which means no new "features" but I'm not sure if it means no bug fixes.

However, there are some other boxplot-specific issues in matplotlib (such as the blue outliers, and strange behavior when you choose different properties) and I'm not sure if they have the same root cause or not.
